packages/components/src/docs/Migration/ui-supernumber.mdx

```diff
@@ -0,0 +1,54 @@
+# Migration Guide: From `ui-supernumber` to `supernumber` pattern
+
+The new `supernumber` pattern is designed to replace the `ui-supernumber`. This transitions away from a component with attributes in favor of a pattern that encourages reusability. It uses a combination of tokens and styles (e.g, sd-paragraph) to create a supernumber. You can take a look at the pattern [here](https://solid-design-system.fe.union-investment.de/x.x.x/storybook/?path=/docs/pattern-supernumber--docs) to use the pattern in your project.
+
+## 💾 Slots
+
+### ❌ Removed Slots
+
+The following slots have been removed from the new `supernumber` pattern:
+
+1. [default]
+
+<hr />
+
+## ⚙️ Attributes
+
+### ❌ Removed Attributes
+
+The following attributes have been removed from the new `supernumber` pattern:
+
+1. [animation-time]
+2. [description]
+3. [increment-number]
+4. [number]
+5. [number-animated]
+6. [number-format]
+7. [number-postfix]
+8. [number-prefix]
+9. [start-number]
+10. [text-after]
+11. [text-before]
+12. [variant]
+13. [width]
+
+<hr />
+## 🥳 Events
+
+### ❌ Removed Events:
+
+The following events have been removed from the new `supernumber` pattern:
```

Comment:
As a pattenr per design can't have those, I would just write a sentence, that therefore no attributes, events or methods are available.